src/components/containers/menu/Menu.js

```diff
@@ -22,23 +28,34 @@ class Menu extends React.Component {
   };
 
   render() {
-    const { children, buttonContent, className } = this.props;
+    const {
+      children,
+      buttonContent,
+      className,
+      theme,
+      openButtonType
+    } = this.props;
 
     return (
-      <div className={className}>
-        <ToggleButton
-          handleOnClick={this.toggleMenu}
-          isPressed={this.state.isMenuOpen}
-        >
-          {buttonContent}
-        </ToggleButton>
-        <MenuPanel
-          isOpen={this.state.isMenuOpen}
-          closeFunction={this.toggleMenu}
-        >
-          {children}
-        </MenuPanel>
-      </div>
+      <ThemeProvider theme={theme}>
```

Comment:
We don't actually need the ThemeProvider here if MenuPanel will always be wrapped by a Menu component.  The Menu ThemeProvider will pass the theme down to all descendant subcomponents.